paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentSheet.kt

```diff
@@ -376,6 +360,7 @@ class PaymentSheet internal constructor(
         val paymentMethodTypes: List<String> = emptyList(),
         val paymentMethodConfigurationId: String? = null,
         val onBehalfOf: String? = null,
+        val requireCvcRecollection: Boolean = false
```

Comment:
This is missing the annotation 🤔 

And adding the annotation here will break everyone using deferred intents. We should probably have a builder for all these things for this reason.

Are we planning to GA right away?

Reply:
Based on discussions on Slack, I think we should just wait to merge until we have GA approval which will hopefully be next week